webpack.config.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
```

Comment:
- [ ] Please don't disable eslint. If it is giving an error kindly solve it.

Reply:
Could you please advise how to resolve eslint issues for auto-generated package files?

Reply:
@petushka1 I open the eslint in this file only giving indentation error.
![image](https://user-images.githubusercontent.com/38635158/175393596-79ca157d-e6af-4cc1-9750-6c2a50cd7e15.png)

- [ ] You can just fix it ctrl+shift+I if you have a prettier extension in the visual studio.

Reply:
Yes, thanks. Nice find. Thanks for the hint. Fixed everything. Thanks for the advice about numerical data. Was thinking about this and gonna extend it for the next iteration or after core requirements completed.